components/locid/src/macros.rs

```diff
@@ -342,78 +340,3 @@ macro_rules! transform_ext_key {
         R
     }};
 }
-
-#[cfg(test)]
-mod test {
```

Comment:
Nit (optional): I know they're redundant but I still have a slight preference for keeping these tests, in part because it makes the code compiled during `cargo check` without needing to build the docs.